tests/parser/features/test_init.py

```diff
@@ -53,3 +53,53 @@ def baz() -> uint8:
 
     n = 256
     assert_compile_failed(lambda: get_contract(code, n))
+
+
+# issue 3292
+def test_internal_functions_called_by_ctor_location(get_contract):
+    code = """
+d: uint256
+x: immutable(uint256)
+
+@external
+def __init__():
+    self.d = 1
+    x = 2
+    self.a()
+
+@external
+def test() -> uint256:
+    return self.d
+
+@internal
+def a():
+    self.d = x
+    """
+    c = get_contract(code)
+    assert c.test() == 2
+
+
+# issue 3206
```

Comment:
```suggestion
# issue 3206 (https://github.com/vyperlang/vyper/issues/3206)
```

See comment above

Reply:
resolved in bbfaf2ab6c17abec9660cfeac76e5db1796da398